helix-view/src/editor.rs

```diff
@@ -174,12 +199,12 @@ impl Editor {
         let doc = Document::default();
         let id = self.documents.insert(doc);
         self.documents[id].id = id;
-        self.switch(id, action);
+        self.switch(id, action, None);
         id
     }
 
-    pub fn open(&mut self, path: PathBuf, action: Action) -> Result<DocumentId, Error> {
-        let path = crate::document::canonicalize_path(&path)?;
+    pub fn open(&mut self, file: File, action: Action) -> Result<DocumentId, Error> {
```

Comment:
Same here, I think adding a new type makes this much more annoying to use than `open` + `jump`.

Reply:
Fixed